python/ray/dag/compiled_dag_node.py

```diff
@@ -1682,11 +1682,17 @@ def _generate_dag_operation_graph_node(
                 method_name = exec_task.method_name
                 actor_handle = dag_node._get_actor_handle()
                 requires_nccl = dag_node.type_hint.requires_nccl()
+                downstream_nccl_actors = (
```

Comment:
Why does `downstream_nccl_actors` check whether the current node requires NCCL while `uspstream_nccl_actors` checks whether the upstream node requires NCCL?